hubblestack/extmods/grains/cloud_details.py

```diff
@@ -35,8 +35,13 @@ def _get_aws_details():
     proxies = {'http': None}
 
     try:
+        ttl_header = {'X-aws-ec2-metadata-token-ttl-seconds': '21600'}
```

Comment:
I think we can reduce TTL value to 5*60 seconds.

Reply:
Done